server/controllers/BooksController.js

```diff
@@ -0,0 +1,32 @@
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { responseMessage } = helpers;
+const { Book, Author } = models;
+
+/**
+ * Add a new book
+ * @name addBooks
+ * @async
+ * @param {Object} request
+ * @param {Object} response
+ * @returns {JSON Object}
+ */
+
+class BooksController {
+  static async addBook(request, response) {
+    const { title, description, isbn, price, yearPublished, stock, authorId } = request.body;
```

Comment:
What if its an author that doesn't exist yet, he wouldn't have an id, let's use author name instead so that we can create an author if the author does not exist

Reply:
done